.virtualenvs/postmkvirtualenv

```diff
@@ -2,6 +2,8 @@
 # This hook is sourced after a new virtualenv is activated.
 
 proj_name=$(echo $VIRTUAL_ENV|awk -F'/' '{print $NF}')
-[ ! -d ~/projects/$proj_name ] && mkdir -p ~/projects/$proj_name
-add2virtualenv ~/projects/$proj_name
-cd ~/projects/$proj_name
+proj_path=$PROJECT_HOME/$new_project_name
```

Comment:
Cleaner. Is `$new_project_name` the variable Virtualenvwrapper sets for this hook? I don't see it existing as a variable name unless it was declared there.